src/components/Wizard/Wizard.test.js

```diff
@@ -42,58 +29,293 @@ test('Wizard loading renders properly', () => {
     </Row>
   );
 
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
+  expect(component.render()).toMatchSnapshot();
 });
 
-test('Wizard embedded renders properly', () => {
-  const onStepClick = jest.fn();
-  const onSidebarItemClick = jest.fn();
-  const activeStepIndex = 0;
-  const activeSubStepIndex = 0;
-  const onNextButtonClick = jest.fn();
-  const onBackButtonClick = jest.fn();
-
-  const component = renderer.create(
-    <Wizard embedded>
+test('Wizard steps renders properly', () => {
+  let eventCount = 0;
+  const component = mount(
+    <Wizard show>
       <Wizard.Header title="Wizard Title" />
       <Wizard.Body>
         <Wizard.Steps
-          steps={renderWizardSteps(mockWizardItems, 0, 0, onStepClick)}
+          steps={[
+            <Wizard.Step
+              key={0}
+              stepIndex={0}
+              step={0}
+              label="Step 1"
+              title="Step 1"
+              onClick={() => eventCount++}
+            >
+              <Wizard.SubStep key={0} subStep="1.1" title="Step 1.1" />
+            </Wizard.Step>
+          ]}
         />
+      </Wizard.Body>
+    </Wizard>
+  );
+
+  component
+    .find('.wizard-pf-step a')
+    .first()
+    .simulate('click');
+
+  expect(eventCount).toBe(1);
+  expect(component.render()).toMatchSnapshot();
```

Comment:
What is this actually testing?  Renders properly doesn't mean much, and the snapshots are almost impossible to either scope the snapshot or add other assertions to each piece.  Examples verify proper step is active, number of steps to match expected steps.  select a single step and verify it's output.

Reply:
many of these tests were rushed - and was just aiming to help get some missing coverage before (in the jest coverage reports printed out in `coverage/lcov-report`). Maybe you can chat w/ @mturley or even share a commit with some more test examples? I started to look at `shallow` for this but ran out of time... so this would be an awesome bonus @dmiller9911.